.github/workflows/build.yaml

```diff
@@ -248,18 +248,20 @@ jobs:
     strategy:
       matrix:
         config:
-          - release
-          - debug
+          - --release
+          - --debug
         compiler:
-          - clang
+          - --clang
     name: MacOS ${{ matrix.config }} ${{ matrix.compiler }}
-    runs-on: macos-10.15  # Building with clang on MacOS 11 is currently broken
+    runs-on: macos-11
     steps:
     - name: Installing dependencies
       run: |
         export ICU_ROOT="$(brew --prefix icu4c)" && \
         export PATH="/usr/local/opt/ccache/libexec:/usr/local/opt/gettext/bin:$PATH";
-        brew install boost ccache glew ninja sdl2 sdl2_image sdl2_mixer sdl2_ttf
+        brew install boost ccache glew ninja sdl2 sdl2_image sdl2_mixer sdl2_ttf dylibbundler
+        brewprefix="`brew --prefix`"
+        export CMAKE_PREFIX_PATH="${brewprefix}/opt/icu4c"
```

Comment:
This line has no effect because changes to environment variables are not preserved between steps

Reply:
You did not address this comment yet :)

Reply:
Well it works without this change, so there is no need to change it?

Reply:
If it's not needed, just remove these 2 lines. But there's no point adding commands that don't do anything

Reply:
Done lets see how we are doing